definitions/ext-firewall/golden_metrics.yml

```diff
@@ -17,6 +17,11 @@ cpuUtilization:
       select: max(kentik.snmp.CPU)
       from: Metric
       where: "provider = 'kentik-firepower'"
+    # Dell Sonicwall firewalls
+    kentik/sonicwall-sma:
+      select: latest(kentik.snmp.CPU)
```

Comment:
this seems to be exactly the same as the one below but using latest instead of max.

I'm wondering if the harvester will create 2 metrics for these ones instead of on for each provider.